argo-messaging.spec

```diff
@@ -0,0 +1,59 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+
+Name: argo-messaging
+Summary: ARGO Messaging API for broker network
+Version: 0.9.1
+Release: 1%{?dist}
+License: ASL 2.0
+Buildroot: %{_tmppath}/%{name}-buildroot
+Group: Unspecified
+Source0: %{name}-%{version}.tar.gz
+BuildRequires: golang
+BuildRequires: git
+ExcludeArch: i386
+Obsoletes: argo-messaging
+
+%description
+Installs the ARGO Messaging API
+
+%prep
+%setup
+
+%build
+export GOPATH=$PWD
+cd src/github.com/ARGOeu/argo-messaging/
+go get github.com/tools/godep
+godep restore
```

Comment:
You need to add the following line right after installing godep and before calling it in subsequent lines:

```
export PATH=$PATH:$GOPATH/bin
godep restore
...
```